src/main/java/com/coherentsolutions/training/auto/web/pashkovskaya/pages/AccountCreationPage.java

```diff
@@ -0,0 +1,35 @@
+package com.coherentsolutions.training.auto.web.pashkovskaya.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AccountCreationPage extends BasePage{
+    @FindBy(id = "firstname")
+    public WebElement firstNameTextField;
+    @FindBy(id = "lastname")
+    public WebElement lastNameTextField;
+    @FindBy(id = "email_address")
+    public WebElement emailTextField;
+    @FindBy(id = "password")
+    public WebElement passwordTextField;
+    @FindBy(id = "password-confirmation")
+    public WebElement passwordConfirmationTextField;
+    @FindBy(xpath = "//button [@class='action submit primary']")
```

Comment:
1) this xpath is ambiguous, let's find more obvious locator
2) don't link your xpath to multiple classes, use just one attribute value